inst/dust/basic.cpp

```diff
@@ -91,8 +91,8 @@ compare(const typename T::real_type * state,
 // [[dust::param(n_trans_classes, has_default = TRUE, default_value = 1L, rank = 0, min = -Inf, max = Inf, integer = FALSE)]]
 class basic {
 public:
-  using real_type = double;
-  using rng_state_type = dust::random::generator<real_type>;
+  typedef double real_type;
```

Comment:
You have an old version of odin.dust or dust - can you upgrade to the most recent version on ncov-ic drat and regenerate?

Reply:
I have done as you suggested, @richfitz  please check if this looks better now? thanks